nucliadb_protos/resources.proto

```diff
@@ -234,6 +234,10 @@ message Paragraph {
     repeated Sentence sentences = 7;
     string key = 8;
     string text = 9;  // Optional, as a computed value
+    uint32 page = 10;
+    bool page_with_table = 11;
+    bool is_a_table = 12;
+    string reference_file = 13;
```

Comment:
@bloodbare can you encapsulate the new fields in a separate message? 
IMO it will be easier to understand the purpose of each field in the Paragraph message.

Reply:
Proposal

```
message TableMetadata {
    uint32 page = 1;
    bool page_with_table = 2;
    bool is_a_table = 3;
    string reference_file = 4;
}

message Paragraph {
  ...
 
  TableMetadata table = 10;
} 


```

Reply:
Its not table metadata only, this requires a call to go deeper